src/system_metrics/service_mongoose_system_metrics.erl

```diff
@@ -0,0 +1,67 @@
+-module(service_mongoose_system_metrics).
+-author('jan.ciesla@erlang-solutions.com').
+
+-behaviour(gen_server).
```

Comment:
Since this module implements `mongoose_service` behaviour, I think it'd be worth to specify it here.

Reply:
In such case, do we add this behaviour or replace current one?

Reply:
Add, it's ok to implement more than one behaviour.